pkg/api.go

```diff
@@ -0,0 +1,85 @@
+package sfu
+
+import (
+	"encoding/json"
+
+	"github.com/pion/ion-sfu/pkg/log"
+	"github.com/pion/webrtc/v3"
+)
+
+// All available data channels commands for SFU, may reflect only changes on caller,
+// this is not intended to force remove,mute, etc. to other publisher tracks.
+// Data channel commands are received in ion-sfu data channel. So it must be considered
+// a reserved name
+const (
```

Comment:
generally i would prefer an api that is constraints oriented, similar to `getUserMedia`. this is because the client has a fixed number of pixels to fill, so it is easier to reason about the track they want by saying "the thumbnail view is 320x240, send me the track with the closest resolution`. otherwise the api is kind of opaque, i dont really know what i am asking for. the resolution logic might be a little complex though

Reply:
Getting the tracks width and height would require inpecting the packets, signaling them is not warranty of the recieved tracks use them this apply more in simulcast/SVC where the substreams are hardcoded.

So if we add a command in client SDK like ```getRemoteMedia(video:"high","medium","low" ,audio:boolean)```, we could just use high, medium or low instead resolutions, and internally translate them to more simple commands for the SFU.

Reply:
i really like that api actually. we could have a single command that accept a payload like:
```
{
     id: "....",
     video:"high"|"medium"|"low"|"none" ,
     audio:boolean
}
```